markets/aave/commons.ts

```diff
@@ -253,6 +255,7 @@ export const CommonsConfig: ICommonConfiguration = {
       SNX: '0x79291A9d692Df95334B1a0B3B4AE6bC606782f8c',
       SUSD: '0x8e0b7e6062272B5eF4524250bFFF8e5Bd3497757',
       TUSD: '0x3886BA987236181D98F2401c507Fb8BeA7871dF2',
+      UMA: '0x04Fa0d235C4abf4BcF4787aF4CF447DE572eF828',
```

Comment:
The address of the CL price feed (ETH denominated): 0xf817B69EA583CAFF291E287CaE00Ea329d22765C

Reply:
Updated to 0xf817B69EA583CAFF291E287CaE00Ea329d22765C